framework/src/meshmodifiers/SideSetsAroundSubdomain.C

```diff
@@ -168,8 +170,9 @@ SideSetsAroundSubdomain::modify()
         const unsigned int side = q.second;
         const Elem * neighbor = elem->neighbor_ptr(side);
 
-        if (neighbor == NULL ||                   // element on boundary OR
-            neighbor->subdomain_id() != block_id) // neighboring element is on a different subdomain
+        if (neighbor == NULL || // element on boundary OR
```

Comment:
Feel free to switch to `nullptr` when you touch lines of code. I want to phase out `NULL`.